src/integrationTest/java/uk/gov/hmcts/reform/professionalapi/CreateNewUserWithRolesTest.java

```diff
@@ -446,32 +446,4 @@ public void validate_invalid_email_for_invite_user_and_throw_exception() {
         assertThat(newUserResponse.get("http_status")).isEqualTo("400");
 
     }
-
-    @Test
-    public void should_return_409_when_same_invited_new_user_is_super_user_of_pending_org() {

```

Comment:
this test seemed to be removed in development branch so I am removing it here too